### PR TITLE
ULTRA-22796 adds methods/tests for actions and getWebElement

### PR DESCRIPTION
**Pull request by casteaux**

### CODE REVIEW

File: test/spec/protractor_sync_test.ts [hunk 1]
```diff
@@ -277,12 +285,18 @@ describe('Protractor extensions', () => {
...
+      }));
+
+      it('automatically resolves promises returned with polledExpect Protractor methods', createTest(() => {
         polledExpect(() => testArea.findElement('.visible-element').getElementFinder().isDisplayed()).toEqual(true);
       }));
 
+      it('works with polled Expect and protractorSync element finder methods on its own', createTest(() => {
+        polledExpect(() => testArea.findElement('.visible-element').isDisplayed()).toEqual(true);
+      }));
+
```
Comment (mindywhitsitt) [inline test/spec/protractor_sync_test.ts]: Not your doing, but these methods don't seem to belong in the findVisible() describe block. None of them are even calling findVisible. :)

Comment (shermaneric) [reply test/spec/protractor_sync_test.ts]: full disclosure the ^ was errantly my doing. good catch mindy.

Comment (casteaux) [reply test/spec/protractor_sync_test.ts]: Good point!

**Review by shermaneric — DISMISSED** [review]
This sounds like a good as place as any for these tests to live.  I ran them just now, and all looks good. Approving.

**Review by mindywhitsitt — DISMISSED** [review]

**Review by mindywhitsitt — APPROVED** [review]

**Review by shermaneric — APPROVED** [review]

**Pull request merged**